packages/server/postgres/queries/src/updateMeetingSeriesByIdQuery.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  @name updateMeetingSeriesByIdQuery
+  @param ids -> (...)
+*/
+UPDATE "MeetingSeries" SET
+  "meetingType" = COALESCE(:meetingType, "meetingType"),
+  "title" = COALESCE(:title, "title"),
+  "recurrenceRule" = COALESCE(:recurrenceRule, "recurrenceRule"),
+  "duration" = COALESCE(:duration, "duration"),
+  "cancelledAt" = COALESCE(:cancelledAt, "cancelledAt")
```

Comment:
TIL: using `COALESCE` will prevent setting `null` value here, will fix in the next PR, this PR isn't affected